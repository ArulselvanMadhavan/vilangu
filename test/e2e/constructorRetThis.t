class A {
  A() { out 42; return this; }
}

int main() {
  A a;
  a = new A();
}