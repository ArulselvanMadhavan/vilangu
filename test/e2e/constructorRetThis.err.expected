./test/e2e/constructorRetThis.t:2:2 - 2:30: constructor/destructor methods should not have a return statement with an expression
